SDWebImage/UIView+WebCache.m

```diff
@@ -48,6 +56,7 @@ - (void)sd_internalSetImageWithURL:(nullable NSURL *)url
     if (!validOperationKey) {
         validOperationKey = NSStringFromClass([self class]);
     }
+    [self setSd_latestOperationKey:validOperationKey];
```

Comment:
Changing this into `self.sd_latestOperationKey = validOperationKey;` looks better :)